frontend/src/chat/components/ChatWindow.vue

```diff
@@ -203,6 +227,14 @@ onMounted(() => {
     subtree: true,
   };
   observer.observe(document.getElementById("chatInput"), config);
+
+  document.getElementById("chatInput").addEventListener("paste", function (e) {
+    e.preventDefault();
+    var text = e.clipboardData.getData("text/plain");
+
+    //execCommand is deprecated, but it's the only way to do this and there is literally no other function with the same functionality...
+    document.execCommand("insertHTML", false, text);
+  });
 });
```

Comment:
This nicely fixes the paste issue.
It removes any formatting of the pasted content. :^)